src/renderer/components/dialogs/Settings-Profile.tsx

```diff
@@ -240,21 +251,30 @@ export function SettingsEditProfileDialogInner({
               setDisplayname(event.target.value)
             }}
           />
-          <DeltaTextarea
-            key='status'
-            id='status'
-            placeholder={tx('pref_default_status_label')}
-            value={selfstatus}
-            onChange={(
-              event: React.FormEvent<HTMLElement> &
-                React.ChangeEvent<HTMLTextAreaElement>
-            ) => {
-              setSelfstatus(event.target.value)
-            }}
-          />
+          {simpleSetup ? (
+            <span>You can change that later in the settings</span>
+          ) : (
```

Comment:
I would just not show the signature field at all in this case, otherwise we would need a really wisely worded string that can be understood by normal users. Here the context is just missing to the user and I don't think it's worth to try to explain it without causing confusion.